src/browser/BrowserAction.cpp

```diff
@@ -406,11 +406,11 @@ QString BrowserAction::getErrorMessage(const int errorCode) const
     case ERROR_KEEPASS_CANNOT_DECRYPT_MESSAGE:
         return QObject::tr("Cannot decrypt message");
     case ERROR_KEEPASS_TIMEOUT_OR_NOT_CONNECTED:
-        return QObject::tr("Timeout or cannot connect to KeePassXC");
+        return QObject::tr("Cannot connect to KeePassXC. Is Browser Integration enabled?");
```

Comment:
Recommend second sentence of: "Check that browser integration is enabled in KeePassXC settings."